plugins/org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/model/GeneratedJavaFileAccess.xtend

```diff
@@ -25,9 +26,17 @@ class GeneratedJavaFileAccess extends JavaFileAccess {
 		if (!markedAsGenerated)
 			throw new IllegalArgumentException("It's always generated");
 	}
-
+	
+	def setTypeComment(StringConcatenationClient javaContent) {
```

Comment:
A question on the return type that is now 'CharSequence'. Is that fine or should I explicitly set it to 'void'?
(did it the same way like in 'JavaFileAccess.setContent(...)')